common-crafting.lic

```diff
@@ -181,4 +181,38 @@ module DRCC
       DRC.bput("stow my #{logbook} logbook", 'You put', 'What were you referring to')
     end
   end
+
+  def order_enchant(stock_room, stock_needed, stock_number, bag, belt)
+    stock_needed.times do
+      order_item(stock_room, stock_number)
+      stow_crafting_item(left_hand, bag, belt)
+      stow_crafting_item(right_hand, bag, belt)
+      next unless left_hand && right_hand
+    end
+  end
+
+  def fount(stock_room, stock_needed, stock_number, quantity, bag, belt)
+    case bput('tap my fount', /You tap .* inside your .*/, /You tap .*your .*/, /I could not find what you were referring to./)
+    when /You tap (.*) inside your (.*)/, /You tap (.*) your (.*)/
+      /(\d+)/ =~ bput('analyze my fount', 'This appears to be a crafting tool and it has approximately \d+ uses remaining')
+      if Regexp.last_match(1).to_i < (quantity + 1)
+        get_tool('fount')
+        dispose('fount')
+        stow_hands
+        order_enchant(stock_room, stock_needed, stock_number)
```

Comment:
❌ These three calls to `order_enchant` also need `bag` and `belt`

Reply:
Added.